samples/QotdService/Program.cs

```diff
@@ -22,9 +22,8 @@ public static void Main(string[] args)
             {
                 connection.ReadCompleted += (data) =>
                 {
-                    var quote = new Utf8String(quoteBytes);
-                    quote.CopyTo(buffer);
-                    connection.TryWrite(buffer, quote.Length);
+                    quote.Bytes.CopyTo(buffer);
```

Comment:
Is there any benefit in caching quote.Bytes into a local? Are we allowed to do it given Bytes is a Span type?

Reply:
In general yes. in this case, it's just a sample. It actually should be rewritten to not use the temporary buffer.